Cargo.lock

```diff
@@ -4,2268 +4,2275 @@
 name = "aho-corasick"
```

Comment:
I think this is all due to an update of the Cargo.lock format ... I'm happy to revert this change (which was unintentional). Not that it is actually used by consumers of this library.